packages/forma-36-website/src/pages/components/radio-button-field/index.mdx

```diff
@@ -5,18 +5,3 @@ status: 'stable'
 github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/RadioButton'
 storybook: 'https://f36-storybook.contentful.com/?path=/story/components-radiobutton--default'
```

Comment:
Looks like these links are to the RadioButton component.

Reply:
yup, thx! updated